vue/vite.config.ts

```diff
@@ -1,35 +1,41 @@
-import { defineConfig } from "vite";
+import { defineConfig, loadEnv } from "vite";
 import vue from "@vitejs/plugin-vue";
 import { execSync } from "child_process";
 
-export default defineConfig(() => {
-  // Load env file based on `mode` in the current working directory.
-  // Set the third parameter to '' to load all env regardless of the `VITE_` prefix.
-  const commitDate = execSync("git log -1 --format=%cI").toString().trimEnd();
-  const branchName = execSync("git rev-parse --abbrev-ref HEAD")
-    .toString()
-    .trimEnd();
-  const commitHash = execSync("git rev-parse HEAD").toString().trimEnd();
-  const lastCommitMessage = execSync("git show -s --format=%s")
-    .toString()
-    .trimEnd();
+export default ({ mode}) => {
+  // Loads all environment variables to see if we are in VITE_DOCKER_DEVELOPMENT
+  process.env = {...process.env, ...loadEnv(mode, process.cwd())};
+    // Load env file based on `mode` in the current working directory.
+    // Set the third parameter to '' to load all env regardless of the `VITE_` prefix.
+    const commitDate = execSync("git log -1 --format=%cI").toString().trimEnd();
+    const branchName = execSync("git rev-parse --abbrev-ref HEAD")
+      .toString()
+      .trimEnd();
+    const commitHash = execSync("git rev-parse HEAD").toString().trimEnd();
+    const lastCommitMessage = execSync("git show -s --format=%s")
+      .toString()
+      .trimEnd();
```

Comment:
Just fixing some inconsistent whitespace -
```suggestion
  // Load env file based on `mode` in the current working directory.
  // Set the third parameter to '' to load all env regardless of the `VITE_` prefix.
  const commitDate = execSync("git log -1 --format=%cI").toString().trimEnd();
  const branchName = execSync("git rev-parse --abbrev-ref HEAD")
    .toString()
    .trimEnd();
  const commitHash = execSync("git rev-parse HEAD").toString().trimEnd();
  const lastCommitMessage = execSync("git show -s --format=%s")
    .toString()
    .trimEnd();
```